File: devconnector/src/main/java/com/devconnector/dto/LikeDTO.java
package com.devconnector.dto;

import com.devconnector.model.Post;
import com.devconnector.model.User;
import lombok.*;

import java.time.Instant;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class LikeDTO {
    private Long id;
    private Instant likedAt;
    private UserDTO userDTO;
    private PostDTO postDTO;
}

File: client/src/app/services/models/like-dto.ts
/* tslint:disable */
/* eslint-disable */
import { PostDto } from '../models/post-dto';
import { UserDto } from '../models/user-dto';
export interface LikeDto {
  id?: number;
  likedAt?: string;
  postDTO?: PostDto;
  userDTO?: UserDto;
}
